vm/devices/storage/storage_tests/tests/scsidvd_nvme.rs

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#![cfg(target_os = "linux")]
```

Comment:
Fascinating. Why does this only run with linux as the target os?

Reply:
Windows hosts (for the unit tests) do not appear to support NVMe. It is this way for other NVMe tests in the codebase.